index.html

```diff
@@ -50,6 +50,141 @@ <h1>I’m Eugene. <br>
+                    </div>
+                </div>
+                <div class="work-container">
+                    <div class="card-container">
+                        <div class="card">
+                            <img src="images/Snapshoot Portfolio (6).svg" alt="">
+                        </div>
+                        <div class="left-block">
+                            <h2>Tonic</h2>
+                            <ul class="position">
+                                <li>CANOPY<i class="fa fa-circle"></i></li>
+                                <li>Back End Dev<i class="fa fa-circle"></i></li>
+                                <li>2015</li>
+                            </ul>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                            <ul class="programming-lang">
+                                <li><a href="#">HTML</a></li>
+                                <li><a href="#">CSS</a></li>
+                                <li><a href="#">Javascript</a></li>
+                            </ul>
+                            <button class="btn-action">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-container">
+                    <div class="card-container">
+                        <div class="card">
+                            <img src="images/Snapshoot Portfolio (7).svg" alt="">
+                        </div>
+                        <div class="left-block">
+                            <h2>Tonic</h2>
+                            <ul class="position">
+                                <li>CANOPY<i class="fa fa-circle"></i></li>
+                                <li>Back End Dev<i class="fa fa-circle"></i></li>
+                                <li>2015</li>
+                            </ul>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                            <ul class="programming-lang">
+                                <li><a href="#">HTML</a></li>
+                                <li><a href="#">CSS</a></li>
+                                <li><a href="#">Javascript</a></li>
+                            </ul>
+                            <button class="btn-action">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        <div class="about-myself">
+           <div class="about-container">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a href="#" class="connect">
+                LET'S CONNECT
+            </a> 
+            <ul class="social-media">
```

Comment:
- [ ] Kindly fix
- [ ] Your design
<img width="520" alt="Screenshot 2023-07-14 at 16 07 17" src="https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81444338/7a36ede0-bb95-4b02-a984-424213107998">

- [ ] Figma design
<img width="358" alt="Screenshot 2023-07-14 at 16 12 58" src="https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81444338/eff4db00-f982-4f18-9fff-a7d39c7214ab">

Reply:
- [ ] Good job making some changes to your design as requested, however, kindly change the color of the paragraph of the `About` section to the one given in the Figma design which is `#344563`

Check the color being used
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/81932478/1faa8932-f6bd-42d4-a796-b80cf65fb9cd)